lte/gateway/python/magma/pipelined/tests/BUILD.bazel

```diff
@@ -0,0 +1,849 @@
+        ":pipelined_test_util",
+        "//lte/gateway/python/magma/pipelined/app:conntrack",
+        "//lte/gateway/python/magma/pipelined/app:testing",
+        "//lte/gateway/python/magma/pipelined/tests/app:packet_injector",
+        "//lte/gateway/python/magma/pipelined/tests/app:start_pipelined",
+        "//lte/gateway/python/magma/pipelined/tests/app:subscriber",
+        "//lte/gateway/python/magma/pipelined/tests/app:table_isolation",
+        "//lte/protos:mconfigs_python_proto",
+    ],
+)
+
+pytest_test(
+    name = "test_cwf_restart_resilience",
+    size = "small",
+    srcs = ["test_cwf_restart_resilience.py"],
+    imports = [
+        LTE_ROOT,
+        ORC8R_ROOT,
+    ],
+    tags = TAG_SUDO_TEST,
+    deps = [
+        ":pipelined_test_util",
+        "//lte/gateway/python/magma/pipelined/app:base",
+        "//lte/gateway/python/magma/pipelined/app:testing",
+        "//lte/gateway/python/magma/pipelined/tests/app:start_pipelined",
+        "//lte/gateway/python/magma/subscriberdb:sid",
+        "//lte/protos:mconfigs_python_proto",
+        "//lte/protos:pipelined_python_proto",
+    ],
+)
+
+pytest_test(
+    name = "test_dpi",
+    size = "small",
+    srcs = ["test_dpi.py"],
+    imports = [
+        LTE_ROOT,
+        ORC8R_ROOT,
+    ],
+    tags = TAG_SUDO_TEST,
+    deps = [
+        ":pipelined_test_util",
+        "//lte/gateway/python/magma/pipelined:bridge_util",
+        "//lte/gateway/python/magma/pipelined:policy_converters",
+        "//lte/gateway/python/magma/pipelined/app:testing",
+        "//lte/gateway/python/magma/pipelined/tests/app:start_pipelined",
+        "//lte/protos:mconfigs_python_proto",
+        "//lte/protos:pipelined_python_proto",
+    ],
+)
+
+pytest_test(
+    name = "test_ebpf_dl_dp",
+    size = "small",
+    srcs = ["test_ebpf_dl_dp.py"],
+    imports = [
+        LTE_ROOT,
+        ORC8R_ROOT,
+    ],
+    tags = TAG_SUDO_TEST,
+    deps = [
+        "//lte/gateway/python/magma/pipelined:bridge_util",
+        "//lte/gateway/python/magma/pipelined/ebpf:ebpf_manager",
+        "//lte/protos:mobilityd_python_proto",
+    ],
+)
+
+pytest_test(
+    name = "test_ebpf_ul_dp",
+    size = "small",
+    srcs = ["test_ebpf_ul_dp.py"],
+    imports = [
+        LTE_ROOT,
+        ORC8R_ROOT,
+    ],
+    tags = TAG_SUDO_TEST,
+    deps = [
+        "//lte/gateway/python/magma/pipelined:bridge_util",
+        "//lte/gateway/python/magma/pipelined/ebpf:ebpf_manager",
+        "//lte/protos:mobilityd_python_proto",
+    ],
+)
+
+pytest_test(
+    name = "test_encoding",
+    size = "small",
+    srcs = ["test_encoding.py"],
+    imports = [LTE_ROOT],
+    deps = [
+        "//lte/gateway/python/magma/pipelined:encoding",
+        "//lte/protos:mconfigs_python_proto",
+    ],
+)
+
+pytest_test(
+    name = "test_enforcement",
+    size = "small",
+    srcs = ["test_enforcement.py"],
+    imports = [
+        LTE_ROOT,
+        ORC8R_ROOT,
+    ],
+    tags = TAG_SUDO_TEST,
+    deps = [
```

Comment:
The `test_enforcement` does not seem to need
```
        "//lte/gateway/python/magma/pipelined/app:he",
        "//lte/gateway/python/magma/pipelined/tests/app:flow_query",
        "//lte/protos:mobilityd_python_proto",
```

Reply:
> The `test_enforcement` does not seem to need
> 
> ```
>         "//lte/gateway/python/magma/pipelined/app:he",
>         "//lte/gateway/python/magma/pipelined/tests/app:flow_query",
>         "//lte/protos:mobilityd_python_proto",
> ```

It can get it transitively and you are right that it could work without them added here, but all 3 of them are imported directly in test_enforcement.py therefore I think they should stay.